graphql_api/types/query/query.py

```diff
@@ -50,11 +52,17 @@ def resolve_owner(_, info, username):
         if not user or not user.is_authenticated:
             raise UnauthorizedGuestAccess()
 
-    return get_owner(service, username)
+        # per product spec, if guestAccess is off for the environment, the current enterpriseUser
```

Comment:
nit: I think the comment states something that can be inferred from the code / PR if needed so not sure if there's much value in having it tbh